proposals/NNNN-New_rules_for_providing_VRHelpItems_VRHelpTitle.md

```diff
@@ -0,0 +1,68 @@
+```
+# Feature name
+
+* Proposal: [SDL-NNNN](NNNN-New_rules_for_providing_VRHelpItems_VRHelpTitle.md)
+* Author: [Irina Getmanets](https://github.com/GetmanetsIrina)
+* Status: **Awaiting review**
+* Impacted Platforms: Core, RPC
+
+## Introduction
+
+VRHelp items and VRHelpTitle are set by SetGlobalProperties for helping user to make voice navigation over registered application, what is very usefull in case your hands are busy with driving.
+
+## Motivation
+
+If there is no SetGlobalProperties issued, the default help item list will be populated by command list shared by Voice module or by default values from .ini file. If App issues a SetGlobalProperties with help item list, then this list will have precedence and it will be displayed on HMI.
+We imagine cases when app does not provide VRHelps item and VRHelpTitle, so in such case help information will be empty.
+The reason of proposal to add SDL logic related to adding values for VRHelp item and VRHelpTitle in case they are not provided by application.
```

Comment:
@GetmanetsIrina , sound a little bit strange for americans
"The reason of proposal to add SDL logic related to adding values for VRHelp item and VRHelpTitle"
to
"The reason of proposal to add logic of adding default values for VRHelp item and VRHelpTitle"

Reply:
Changed locally to "The reason of proposal to add logic of adding values for VRHelp item and VRHelpTitle"